django/tests/text_extractor/helpers/test_views.py

```diff
@@ -73,9 +73,9 @@ def test_document_submission_and_download(client, all_apps_user, mock_pdf_file):
     docs = response.context["ocr_docs"]
     assert len(docs) == 1
 
-    # There should be two properties on the doc: "pdf" and "txt". Each has a file_id
-    pdf_file_id = docs[0]["pdf"]["file"].file_id
-    txt_file_id = docs[0]["txt"]["file"].file_id
+    # There should be two properties on the doc: "pdf" and "txt". Each has an ID.
+    pdf_file_id = docs[0]["pdf"]["file"].id
+    txt_file_id = docs[0]["txt"]["file"].id
```

Comment:
In a recently merged PR, the "file_id" field was removed from the object.

Reply:
tests working fine now.